docs/guides/exporters/index.md

```diff
@@ -1,103 +1,218 @@
+### Push to registry
 
-To use any of the exporters, you need to specify it by either using the
-available shorthands, or passing the exporter name and its parameters to the
-[`--output`](../../reference/buildx_build.md#output) flag.
+To push a built image to a container registry, you can use the `registry` or
+`image` exporters.
 
-To get the full flexibility out of the various exporters buildkit has to offer,
-you'll need to use the full form of the `--output` flag. However, for
-ease-of-use, buildx also offers various familiar short-hands.
+When you pass the `--push` option to the Buildx CLI, you instruct BuildKit to
+push the built image to the specified registry:
 
-### Pushing images
+```console
+$ docker buildx build --tag <registry>/<image> --push .
+```
 
-You may already be familiar with the following `buildx` command syntax using
-the `-t`/`--tag` and `--push` shorthands to push the resulting image to a
-registry:
+Under the hood, this uses the `image` exporter, and sets the `push` parameter.
+It's the same as using the following long-form command using the `--output`
+option:
 
 ```console
-$ docker buildx build . --tag <registry>/<image> --push
+$ docker buildx build \
+  --output type=image,name=<registry>/<image>,push=true .
 ```
 
-You can replicate this command without the shorthands, using the full
-`--output` flag:
+You can also use the `registry` exporter, which does the same thing:
 
 ```console
-$ docker buildx build . --output type=image,name=<registry>/<image>,push=true
+$ docker buildx build \
+  --output type=registry,name=<registry>/<image> .
 ```
 
-### Loading images
+### Export image layout to file
 
-Images built using the [Docker "default" Buildx driver](../../guides/drivers/docker.md)
-will be automatically loaded into the engine. This means you can run them
-immediately, and they'll be visible in the `docker images` view.
+You can use either the `oci` or `docker` exporters to save the build results to
+image layout on your local filesystem. Both of these exporters generate a tar
+archive file containing the corresponding image layout. The `dest` parameter
+defines the target output path for the tarball.
 
-Images built using any of theh other drivers will not be automatically
-loaded, and instead require manually adding the `--load` flag:
+```console
+$ docker buildx build --output type=oci,dest=./myimage .
+[+] Building 0.8s (7/7) FINISHED
+ ...
+ => exporting to oci image format                                                                     0.0s
+ => exporting layers                                                                                  0.0s
+ => exporting manifest sha256:c1ef01a0a0ef94a7064d5cbce408075730410060e253ff8525d1e5f7e27bc900        0.0s
+ => exporting config sha256:eadab326c1866dd247efb52cb715ba742bd0f05b6a205439f107cf91b3abc853          0.0s
+ => sending tarball                                                                                   0.0s
+$ tar -xf ./myimage
```

Comment:
Let's unpack this into a separate directory, and then `tree` that directory, so we don't have it mixed up with the `Dockerfile`.